.github/workflows/sentry-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Create Sentry Release
+
+on:
+  push:
+    branches:
+      - 'trunk'
+
+jobs:
+  build:
+  - uses: actions/checkout@v2
+  - name: Create Sentry release
+    uses: getsentry/action-release@v1
+    env:
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: a8c
+      SENTRY_PROJECT: calypso
+    with:
+      finalize: false
+      version_prefix: calypso_
```

Comment:
it uses the sha by default for the normal version, and we have it prefixed with `calypso_` elsewhere to make it easy to pick out in sentry.